release/mixer.sh

```diff
@@ -78,20 +78,26 @@ build_update() {
 }
 
 build_deltas() {
+    local mix_format="${1}"
+
     section "Deltas"
 
-    log "Building delta packs..." 1
-    if (( NUM_DELTA_BUILDS > 0 )); then
-        sudo_mixer_cmd build delta-packs --previous-versions "${NUM_DELTA_BUILDS}"
+    local first_file="update/www/version/format${mix_format}/first"
+    if [[ -s "${first_file}" ]]; then
+        local first_build="$(< "${first_file}")"
+        log "Building deltas from the first build ${first_build} in format ${mix_format}..."
+        sudo_mixer_cmd build delta-packs --from "${first_build}"
+        sudo_mixer_cmd build delta-manifests --from "${first_build}"
     else
-        log "Skipping delta pack creation" "No positive number of build were specified"
+        log "Skipping delta creation from the first build ${first_build} in format ${mix_format}" "The first file ${first_file} does not exist"
```

Comment:
I would simplify the msg:

log "Skipping delta for first version" "'first' file not found for '${mix_format}'"